bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/scheduling/Scheduler.java

```diff
@@ -509,13 +491,6 @@ private void addToRunnableList(SchedulerItem item, ItemGroup group) {
 
     public FutureValue createFuture(Strand parent, Callback callback, Map<String, Object> properties,
                                     Type constraint, String name, StrandMetadata metadata) {
-        Strand newStrand = new Strand(name, metadata, this, parent, properties);
```

Comment:
Shall we remove this Strand constructor as well. I think that don't have any usages.